<!DOCTYPE html>
<html lang="zh">
    <head>
  <!-- 元数据 -->
  <meta charset="utf-8">
  
    <link rel="icon" href="/myblog.github.io/assets/manbo.png">
  
  
  <title>AI Multiomics | Hammerbrother的博客</title>
  <link rel="canonical" href="https://hyperion1230.github.io/myblog.github.io/posts/AI-Multiomics/">
  <meta name="author" content="Hammerbrother" />
  <meta http-equiv="Cache-Control" content="no-transform" />
  <meta http-equiv="Cache-Control" content="no-siteapp" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta name="robots" content="index,follow" />
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
  <meta name="format-detection" content="telphone=no, email=no" />
  
    <meta name="keywords" content="Multiomics, Single cell" />
  
  <meta name="description" content="单细胞多组学技术实践 本文档主要分为实践与理论两个部分，在实践部分会post上部分代码，建议阅读顺序为先理论后实践！  理论分析  知识点： Gene activity matrix：基因活性矩阵，是对 scATAC-seq 峰的计数进行总结，利用观察到的基因启动子和增强子处的reads，代表一个合成的 scRNA-seq 数据集，以便用于后续整合。（间接性与细胞异质性） Multimodal">
<meta property="og:type" content="article">
<meta property="og:title" content="AI Multiomics">
<meta property="og:url" content="https://hyperion1230.github.io/myblog.github.io/posts/AI-Multiomics/index.html">
<meta property="og:site_name" content="Hammerbrother的博客">
<meta property="og:description" content="单细胞多组学技术实践 本文档主要分为实践与理论两个部分，在实践部分会post上部分代码，建议阅读顺序为先理论后实践！  理论分析  知识点： Gene activity matrix：基因活性矩阵，是对 scATAC-seq 峰的计数进行总结，利用观察到的基因启动子和增强子处的reads，代表一个合成的 scRNA-seq 数据集，以便用于后续整合。（间接性与细胞异质性） Multimodal">
<meta property="og:locale">
<meta property="og:image" content="https://hyperion1230.github.io/myblog.github.io/posts/AI-Multiomics/image-20251022132956071.png">
<meta property="article:published_time" content="2025-09-22T10:53:05.000Z">
<meta property="article:modified_time" content="2025-10-22T07:14:55.299Z">
<meta property="article:author" content="Hammerbrother">
<meta property="article:tag" content="Multiomics">
<meta property="article:tag" content="Single cell">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://hyperion1230.github.io/myblog.github.io/posts/AI-Multiomics/image-20251022132956071.png">
  
    <link rel="alternate" href="/myblog.github.io/atom.xml" title="Hammerbrother的博客" type="application/atom+xml">
  
  <!-- 样式表文件 -->
  <link rel="stylesheet" id="kratos-css" href="/myblog.github.io/css/kr-core.min.css" media="all"></link>
  
    <link rel="stylesheet" id="highlight-css" href="/myblog.github.io/css/highlight/highlight.js/night-eighties.min.css" media="all"></link>
  
  <link rel="stylesheet" id="theme-light-css" href="/myblog.github.io/css/kr-theme/light.min.css" media="all"></link>
  <link rel="stylesheet" id="theme-dark-css" href="/myblog.github.io/css/kr-theme/dark.min.css" media="(prefers-color-scheme: dark)"></link>
  <script src="/myblog.github.io/js/kr-theme.min.js"></script>
  <link rel="stylesheet" id="fontawe-css" href="/myblog.github.io/vendors/font-awesome@4.7.0/css/font-awesome.min.css" media="all"></link>
  
    <link rel="stylesheet" id="nprogress-css" href="/myblog.github.io/vendors/nprogress@0.2.0/nprogress.css" media="all"></link>
  
  
    <link rel="stylesheet" href="/myblog.github.io/vendors/viewerjs@1.11.6/dist/viewer.min.css"></link>
  
  <!-- 不得不预先加载的一些JS文件 -->
  
    <script src="/myblog.github.io/vendors/qr-code-styling@1.6.0-rc.1/lib/qr-code-styling.js"></script>
  
  <!-- 自定义站点横幅和背景 -->
  <style>
    
      .kratos-cover.kratos-cover-2 {
        background-image: url('/myblog.github.io/images/ban.png');
      }
      html[data-theme="dark"] .kratos-cover.kratos-cover-2 {
        background-image: url('/myblog.github.io/images/ban-dark.png');
      }
    
    
      @media(min-width:768px) {
        body.custom-background {
          background-image: url('/myblog.github.io/images/background-sun.png');
        }
        html[data-theme="dark"] body.custom-background {
          background-image: url('/myblog.github.io/images/background.png');
        }
      }
    
    
      .widget-kratos-about .photo-background {
        background-image: url('/images/default.webp');
      }
      html[data-theme="dark"] .widget-kratos-about .photo-background {
        background-image: url('/images/default.webp');
      }
    
  </style>

  <!-- 额外的追加注入项 -->
  
    <link rel="stylesheet" href="/myblog.github.io/css/custom.css">
<link rel="stylesheet" href="/myblog.github.io/music/APlayer.min.css">
<link rel="stylesheet" href="https://unpkg.com/gitalk/dist/gitalk.css">
<link rel="stylesheet" href="/myblog.github.io/comments/gitalk.css" />
<link rel="stylesheet" href="/myblog.github.io/music/aplayer.css">

  
<meta name="generator" content="Hexo 8.0.0"></head>

    <body class="custom-background">
        <div id="kratos-wrapper">
    <div id="kratos-page">
        <div id="kratos-header">
            <header id="kratos-desktop-topnav" class="kratos-topnav">
                <div class="container">
                    <div class="nav-header">
                        <nav id="kratos-menu-wrap">
                            <ul id="kratos-primary-menu" class="sf-menu">
                                
                                    <li>
                                        
                                            <a href="/myblog.github.io/">
                                        
                                            
                                                <i class="fa fa-home"></i>
                                            
                                            Home
                                        </a>
                                        
                                    </li>
                                
                                    <li>
                                        
                                            <a href="/myblog.github.io/archives/">
                                        
                                            
                                                <i class="fa fa-archive"></i>
                                            
                                            Archives
                                        </a>
                                        
                                    </li>
                                
                                    <li>
                                        
                                            <a>
                                        
                                            
                                                <i class="fa fa-link"></i>
                                            
                                            Links
                                        </a>
                                        
                                            <ul class="sub-menu">
                                                
                                                    <li>
                                                        <a target="_blank" rel="noopener" href="https://github.com/Hyperion1230">
                                                            
                                                                <i class="fa fa-code"></i>
                                                            
                                                            github
                                                        </a>
                                                    </li>
                                                
                                                    <li>
                                                        <a target="_blank" rel="noopener" href="https://wiki.krt.moe">
                                                            
                                                                <i class="fa fa-book"></i>
                                                            
                                                            Docs
                                                        </a>
                                                    </li>
                                                
                                            </ul>
                                        
                                    </li>
                                
                                    <li>
                                        
                                            <a>
                                        
                                            
                                                <i class="fa fa-gamepad"></i>
                                            
                                            Relax
                                        </a>
                                        
                                            <ul class="sub-menu">
                                                
                                                    <li>
                                                        <a href="/myblog.github.io/game/2048/index.html">
                                                            
                                                                <i class="fa fa-trophy"></i>
                                                            
                                                            2048
                                                        </a>
                                                    </li>
                                                
                                                    <li>
                                                        <a href="/myblog.github.io/game/ball/index.html">
                                                            
                                                                <i class="fa fa-futbol-o"></i>
                                                            
                                                            Ball
                                                        </a>
                                                    </li>
                                                
                                                    <li>
                                                        <a href="/myblog.github.io/game/maze/index.html">
                                                            
                                                                <i class="fa fa-map-marker"></i>
                                                            
                                                            Maze
                                                        </a>
                                                    </li>
                                                
                                            </ul>
                                        
                                    </li>
                                
                            </ul>
                        </nav>
                    </div>
                </div>
            </header>
            <header id="kratos-mobile-topnav" class="kratos-topnav">
                <div class="container">
                    <div class="color-logo"><a href="/myblog.github.io/">Hammerbrother的博客</a></div>
                    <div id="kratos-nav-toggle-wrapper" class="nav-toggle">
                        <a id="kratos-nav-toggle" class="kratos-nav-toggle">
                            <i></i>
                        </a>
                    </div>
                </div>
            </header>
        </div>
        <div class="kratos-start kratos-hero-2">
            <!-- <div class="kratos-overlay"></div> -->
            <div class="kratos-cover kratos-cover-2 text-center">
                <div class="desc desc2 animate-box">
                    <a href="/myblog.github.io/">
                        <h2>Hammerbrother的博客</h2> <br />
                        <span>欢迎一起交流进步</span>
                    </a>
                </div>
            </div>
        </div>

        <div id="kratos-blog-post">
            <div class="container">
                <div id="main" class="row">


        

            

            <section class="kr-main-col col-md-8">

        

            <article itemscope itemtype="https://schema.org/Article">
    <link itemprop="mainEntityOfPage" href="https://hyperion1230.github.io/myblog.github.io/posts/AI-Multiomics/">
    <div class="kratos-hentry kratos-page-inner clearfix">
        <header class="kratos-page-header">
            
                <h1 class="kratos-page-title text-center" itemprop="name headline">AI Multiomics</h1>
            
            <ul class="kratos-page-meta text-center">
                <li><time datetime="2025-09-22T10:53:05.000Z" itemprop="datePublished"><i class="fa fa-calendar"></i> 2025-09-22</time></li>
                <li itemprop="author" itemscope itemtype="https://schema.org/Person">
                    <i class="fa fa-user"></i> 作者 <span itemprop="name">Hammerbrother</span>
                </li>
                
                    <li>
                        <i class="fa fa-edit"></i> 
                        
                        
                            ~9.34K
                        
                        字
                    </li>
                
                
                
            </ul>
        </header>
        <div class="kratos-page-content kr-post">
            
            
            
                <div class="kratos-post-inner-toc toc-div-class" >
                    <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#%E5%8D%95%E7%BB%86%E8%83%9E%E5%A4%9A%E7%BB%84%E5%AD%A6%E6%8A%80%E6%9C%AF%E5%AE%9E%E8%B7%B5"><span class="toc-number">1.</span> <span class="toc-text"> 单细胞多组学技术实践</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#%E7%90%86%E8%AE%BA%E5%88%86%E6%9E%90"><span class="toc-number">1.1.</span> <span class="toc-text"> 理论分析</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#%E7%9F%A5%E8%AF%86%E7%82%B9"><span class="toc-number">1.1.1.</span> <span class="toc-text"> 知识点：</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%E9%85%8D%E5%AF%B9%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95"><span class="toc-number">1.1.2.</span> <span class="toc-text"> 配对数据分析方法</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%E9%9D%9E%E9%85%8D%E5%AF%B9%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95%E9%87%8D%E7%82%B9"><span class="toc-number">1.1.3.</span> <span class="toc-text"> 非配对数据分析方法（重点）</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%E8%AF%A5%E9%A2%86%E5%9F%9F%E7%A0%94%E7%A9%B6%E8%BF%9B%E5%B1%95"><span class="toc-number">1.1.4.</span> <span class="toc-text"> 该领域研究进展</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#%E7%9B%AE%E5%89%8D%E6%80%9D%E8%B7%AF"><span class="toc-number">1.2.</span> <span class="toc-text"> 目前思路</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#%E6%B5%81%E7%A8%8B"><span class="toc-number">1.2.0.1.</span> <span class="toc-text"> 流程</span></a><ol class="toc-child"><li class="toc-item toc-level-5"><a class="toc-link" href="#1%EF%B8%8F%E2%83%A3%E6%95%B0%E6%8D%AE%E5%87%86%E5%A4%87"><span class="toc-number">1.2.0.1.1.</span> <span class="toc-text"> 1️⃣数据准备</span></a></li></ol></li><li class="toc-item toc-level-4"><a class="toc-link" href="#%E5%85%B3%E9%94%AE"><span class="toc-number">1.2.0.2.</span> <span class="toc-text"> 关键</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#%E6%95%B0%E6%8D%AE%E8%BD%BD%E5%85%A5"><span class="toc-number">1.2.0.3.</span> <span class="toc-text"> 数据载入</span></a></li></ol></li></ol></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#reference"><span class="toc-number">2.</span> <span class="toc-text"> Reference</span></a></li></ol>
                </div>
            
            <hr />
            <div itemprop="articleBody"><h1 id="单细胞多组学技术实践"><a class="markdownIt-Anchor" href="#单细胞多组学技术实践"></a> 单细胞多组学技术实践</h1>
<p>本文档主要分为实践与理论两个部分，在实践部分会post上部分代码，建议阅读顺序为先理论后实践！</p>
<h2 id="理论分析"><a class="markdownIt-Anchor" href="#理论分析"></a> 理论分析</h2>
<h3 id="知识点"><a class="markdownIt-Anchor" href="#知识点"></a> 知识点：</h3>
<p><code>Gene activity matrix</code>：基因活性矩阵，是对 scATAC-seq 峰的计数进行总结，利用观察到的基因启动子和增强子处的reads，代表一个合成的 scRNA-seq 数据集，以便用于后续整合。（间接性与细胞异质性）</p>
<p><code>Multimodal measurements</code>：多模态测量，可以在同一细胞中探测不同的分子特征，目前较少数据。</p>
<p><code>Manifold</code>: 流形,指高维数据在低维空间中呈现的潜在几何结构。</p>
<p><code>ARI(Adjusted Rand Index)</code>: 兰德指数，<strong>衡量聚类结果与真实类别标签一致性的常用评价指标</strong>，核心作用是量化 “聚类划分” 与 “已知真实分类” 之间的吻合程度，尤其适用于单细胞数据分析、机器学习等需要评估聚类效果的场景。</p>
<hr>
<p>由于配对数据的获得比较（<span class="blur">非常</span>）困难，与此同时我们现有大量的相似或者相近的非配对数据存在，因此对于这部分数据的利用在现在看来尤为关键！</p>
<h3 id="配对数据分析方法"><a class="markdownIt-Anchor" href="#配对数据分析方法"></a> 配对数据分析方法</h3>
<table>
<thead>
<tr>
<th style="text-align:left">Algorithm算法</th>
<th style="text-align:left">Language语言</th>
<th style="text-align:left">Modality and technology模态与技术</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">MOFA+ [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0041">41</a>]</td>
<td style="text-align:left">R</td>
<td style="text-align:left">RNA expression + DNA methylation + chromatin <strong>accessibility (scNMT-seq) RNA 表达 + DNA 甲基化 + 染色质可及性（scNMT-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">BABEL [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0042">42</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (10× Genomics; SNARE-seq; SHARE-seq); RNA expression + protein epitope (CITE-seq) <strong>RNA 表达+染色质可及性（10× Genomics；SNARE-seq；SHARE-seq）；RNA 表达+蛋白表位（CITE-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">Seurat v4 [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0043">43</a>]</td>
<td style="text-align:left">R</td>
<td style="text-align:left">RNA expression + protein epitope (CITE-seq) <strong>RNA 表达+蛋白表位（CITE-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">scAI [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0044">44</a>]</td>
<td style="text-align:left">MATLAB</td>
<td style="text-align:left">RNA expression + chromatin accessibility (sci-CAR); <strong>RNA 表达+染色质可及性（sci-CAR）；</strong></td>
</tr>
<tr>
<td style="text-align:left">R</td>
<td style="text-align:left">RNA expression + DNA methylation (scM&amp;T-seq) RNA 表达+DNA 甲基化（scM&amp;T-seq）</td>
<td style="text-align:left"></td>
</tr>
<tr>
<td style="text-align:left">DCCA [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0045">45</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + DNA methylation (scNMT-seq); scATAC-seq + scRNA-seq (SNARE-seq; SHARE-seq; 10× Genomics) <strong>RNA 表达 + DNA 甲基化（scNMT-seq）；scATAC-seq + scRNA-seq（SNARE-seq；SHARE-seq；10× Genomics）</strong></td>
</tr>
<tr>
<td style="text-align:left">scDEC [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0046">46</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (10× Genomics) <strong>RNA 表达+染色质可及性（10× Genomics）</strong></td>
</tr>
<tr>
<td style="text-align:left">Schema [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0047">47</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (sci-CAR; 10× Genomics) <strong>RNA 表达+染色质可及性（sci-CAR；10× Genomics）</strong></td>
</tr>
<tr>
<td style="text-align:left">scMVAE [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0048">48</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (SNARE-seq; scCAT-seq) <strong>RNA 表达+染色质可及性（SNARE-seq；scCAT-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">DAVAE [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0049">49</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (10× Genomics) <strong>RNA 表达+染色质可及性（10× Genomics）</strong></td>
</tr>
<tr>
<td style="text-align:left">NIC [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0050">50</a>]</td>
<td style="text-align:left">MATLAB</td>
<td style="text-align:left">RNA expression + chromatin accessibility <strong>RNA 表达+染色质可及性</strong></td>
</tr>
<tr>
<td style="text-align:left">scMVP [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0050">50</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (sci-CAR; paired-seq; SNARE-seq; SHARE-seq) <strong>RNA 表达+染色质可及性（sci-CAR；paired-seq；SNARE-seq；SHARE-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">SMILE [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0052">52</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (SNARE-seq; sci-CAR; SHARE-seq); DNA methylation + chromosome structure data <strong>RNA 表达 + 染色质可及性（SNARE-seq；sci-CAR；SHARE-seq）；DNA 甲基化 + 染色体结构数据</strong></td>
</tr>
</tbody>
</table>
<h3 id="非配对数据分析方法重点"><a class="markdownIt-Anchor" href="#非配对数据分析方法重点"></a> <mark>非配对数据分析方法</mark>（重点）</h3>
<table>
<thead>
<tr>
<th style="text-align:left">Alogorithm算法</th>
<th style="text-align:left">Language语言</th>
<th style="text-align:left">Modality and technology模态与技术</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">GULER [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0051">51</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (sci-CAR; SNARE-seq; scATAC-seq) <strong>RNA 表达+染色质可及性（sci-CAR；SNARE-seq；scATAC-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">MATCHER [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0053">53</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility; RNA expression + DNA methylation(scM&amp;T-seq; sc-GEM); RNA expression + chromatin immunoprecipitation sequence <strong>RNA 表达 + 染色质可及性；RNA 表达 + DNA 甲基化（scM&amp;T-seq；sc-GEM）；RNA 表达 + 染色质免疫沉淀测序</strong></td>
</tr>
<tr>
<td style="text-align:left">Seurat v3 [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0054">54</a>]</td>
<td style="text-align:left">R</td>
<td style="text-align:left">RNA expression + chromatin accessibility (scRNA-seq + scATAC-seq; SMART-seq2 + scATAC-seq; SMART-seq2 + sci-ATAC-seq1) <strong>RNA 表达+染色质可及性（scRNA-seq+scATAC-seq；SMART-seq2+scATAC-seq；SMART-seq2+sci-ATAC-seq1）</strong></td>
</tr>
<tr>
<td style="text-align:left">Liger [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0055">55</a>]</td>
<td style="text-align:left">R</td>
<td style="text-align:left">RNA expression + DNA methylation; <strong>RNA 表达 + DNA 甲基化；</strong></td>
</tr>
<tr>
<td style="text-align:left">CoupleNMF [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0031">31</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (SMART-seq + scATAC-seq) <strong>RNA 表达+染色质可及性（SMART-seq+scATAC-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">scACE [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0056">56</a>]</td>
<td style="text-align:left">R</td>
<td style="text-align:left">RNA expression + chromatin accessibility <strong>RNA 表达+染色质可及性</strong></td>
</tr>
<tr>
<td style="text-align:left">scAMACE [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0057">57</a>]</td>
<td style="text-align:left">R、Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility <strong>RNA 表达+染色质可及性</strong></td>
</tr>
<tr>
<td style="text-align:left">scGCN [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0058">58</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (sci-CAR; 10× Genomics; scRNA-seq + sci-ATAC-seq) <strong>RNA 表达+染色质可及性（sci-CAR；10× Genomics；单细胞 RNA 测序+单细胞 ATAC 测序）</strong></td>
</tr>
<tr>
<td style="text-align:left">UnionCom [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0059">59</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + DNA methylation (sc-GEM); RNA expression + DNA methylation + chromatin accessibility (scNMT-seq) <strong>RNA 表达+DNA 甲基化（sc-GEM）；RNA 表达+DNA 甲基化+染色质可及性（scNMT-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">CoupleCoC [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0060">60</a>]</td>
<td style="text-align:left">MATLAB</td>
<td style="text-align:left">RNA expression + chromatin accessibility (scATAC-seq + scRNA-seq; sci-ATAC-seq + scRNA-seq); RNA expression + DNA methylation (scRNA-seq + snmC-seq) <strong>RNA 表达+染色质可及性（scATAC-seq + scRNA-seq；sci-ATAC-seq + scRNA-seq）；RNA 表达+DNA 甲基化（scRNA-seq + snmC-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">MMD-MA [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0061">61</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + DNA methylation (scM&amp;T-seq) <strong>RNA 表达+DNA 甲基化（scM&amp;T-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">MultiMAP [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0062">62</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility <strong>RNA 表达+染色质可及性</strong></td>
</tr>
<tr>
<td style="text-align:left">scJoint [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0063">63</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (scRNA-seq + sci-ATAC-seq; scRNA-seq + scATAC-seq; SNARE-seq); multimodal data (CITE-seq + ASAP-seq) <strong>RNA 表达+染色质可及性（scRNA-seq+sci-ATAC-seq；scRNA-seq+scATAC-seq；SNARE-seq）；多模态数据（CITE-seq+ASAP-seq）</strong></td>
</tr>
<tr>
<td style="text-align:left">Pamona [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0064">64</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + DNA methylation (sc-GEM); RNA expression + DNA methylation + chromatin accessibility (scNMT-seq); RNA expression + chromatin accessibility (SNARE-seq; 10× Genomics) <strong>RNA 表达+DNA 甲基化（sc-GEM）；RNA 表达+DNA 甲基化+染色质可及性（scNMT-seq）；RNA 表达+染色质可及性（SNARE-seq；10× Genomics）</strong></td>
</tr>
<tr>
<td style="text-align:left">uniPort [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0065">65</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility (scRNA-seq + scATAC-seq) <strong>RNA 表达+染色质可及性（单细胞 RNA 测序+单细胞 ATAC 测序）</strong></td>
</tr>
<tr>
<td style="text-align:left">VIPCCA [<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0066">66</a>]</td>
<td style="text-align:left">Python</td>
<td style="text-align:left">RNA expression + chromatin accessibility + DNA methylation (scNMT-seq; scRNA-seq + sci-ATAC-seq + sc-methylation data) <strong>RNA 表达+染色质可及性+DNA 甲基化（单细胞 NMT 测序；单细胞 RNA 测序+sci-ATAC 测序+单细胞甲基化数据）</strong></td>
</tr>
</tbody>
</table>
<blockquote>
<p>from <a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0018">https://onlinelibrary.wiley.com/doi/10.1002/qub2.91#qub291-bib-0018</a></p>
</blockquote>
<div class="alert alert-info">
  <div class="icon"><i class="fa fa-info"></i></div>
  <div class="text">
  <p>接下来我们重点关注的是非配对数据的整合</p>

  </div>
  </div>
<h3 id="该领域研究进展"><a class="markdownIt-Anchor" href="#该领域研究进展"></a> 该领域研究进展</h3>
<blockquote>
<p>单细胞 RNA 测序（scRNA-seq）测量单个细胞中的基因表达，而单核 ATAC 测序（snATAC-seq）量化单个核中的染色质可及性。这两种数据类型提供了解码细胞类型和状态的信息。然而，当单独分析时，它们有时会在细胞类型/状态分配方面产生相互矛盾的结果。由于这两种模态反映了相同的潜在生物学，因此其能力受到损害。最近，从同一核中测量基因表达和染色质可及性已成为可能。这种成对数据使得可以直接建模两种模态之间的关系。鉴于大量单模态数据的可用性，将成对和无成对的单模态数据集整合起来，以获得对细胞复杂性的全面了解，是非常有吸引力的。</p>
<p><a target="_blank" rel="noopener" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03073-x">基准算法：无配对和配对单细胞 RNA 测序与 ATAC 测序数据的联合整合 | 基因组生物学 | 完整内容 — Benchmarking algorithms for joint integration of unpaired and paired single-cell RNA-seq and ATAC-seq data | Genome Biology | Full Text</a></p>
</blockquote>
<p>现有的整合来自非同一批次细胞的数据的方法大致分为以下四类：</p>
<ul>
<li><strong>流型对齐</strong></li>
<li><strong>矩阵分解（Liger ，coupledNMF）</strong></li>
<li><strong>跨模态领近细胞相关性（Seurat）</strong></li>
<li><strong>NN</strong></li>
</ul>
<p>对于第一第三种方法，不适合于过大的数据集整合，第二种方法具有较好的准确度但是对于计算量的要求也非常大，第四种方法比较折中，是目前较多考虑的方法。</p>
<p>下图为GLUE的测试结果：</p>
<p><img src="/myblog.github.io/posts/AI-Multiomics/image-20251022132956071.png" alt="image-20251022132956071"></p>
<hr>
<h2 id="目前思路"><a class="markdownIt-Anchor" href="#目前思路"></a> 目前思路</h2>
<h4 id="流程"><a class="markdownIt-Anchor" href="#流程"></a> 流程</h4>
<h5 id="1️⃣数据准备"><a class="markdownIt-Anchor" href="#1️⃣数据准备"></a> 1️⃣数据准备</h5>
<p>对于单细胞转录组（scRNA-seq）数据而言，其得到的原始数据矩阵是包含了很多系统噪音与随机性噪音的，为了减少这部分噪音对生物生物信号的掩盖，我们需要对这部分数据进行整理，主要包括以下步骤</p>
<p>1 质量控制</p>
<blockquote>
<p>大多数分析任务假设每个液滴都含有来自完整单个细胞的 RNA。这一假设常因细胞质量差、游离 RNA 污染或捕获多个细胞而被违反。检测到的<strong>基因数量少</strong>、<strong>计数深度低</strong>以及<strong>线粒体计数比例高</strong>的细胞通常被称为<strong>低质量细胞</strong>，因为它们可能代表膜破裂的死亡细胞。</p>
</blockquote>
<p>通过手动设置阈值或基于中位数绝对偏差数量的样本特异性自动过滤来识别和过滤。这些指标被综合考虑，以防止对细胞信号的误判。<strong>质量控制是在样本水平上进行的，因为阈值在不同样本之间可能差异很大。</strong></p>
<figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#TODO</span></span><br></pre></td></tr></table></figure>
<blockquote>
<p><strong>游离 RNA</strong> 可能存在于细胞溶液中，并在文库构建过程中被分配到细胞的原生 RNA 中。环境 RNA 污染会导致细胞类型特异性标记基因转录本在其他细胞群体中也可检测到，从而将不同的细胞群体混合在一起。<strong>空滴和双滴</strong>（含有两个细胞的滴）违反了每个滴只含有单个细胞的假设。由不同细胞类型形成的双滴（异型双滴）难以注释，并可能导致错误的细胞类型标签。常见的双滴检测方法通过组合两个随机采样的细胞，并将它们与测量的细胞进行比较来生成人工双滴。</p>
</blockquote>
<figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#TODO</span></span><br></pre></td></tr></table></figure>
<p>在对数据进行初步的质量控制后，我们需要进一步的对测序过程与细胞生物学差异带来的随机性进行矫正。</p>
<blockquote>
<p>细胞由于 mRNA 含量差异（细胞大小）或测序过程中的纯随机性，可能具有不同数量的基因计数。计数标准化使细胞特征具有可比性。后续的方差稳定化确保异常值特征对整体数据结构的影响有限。</p>
<p><strong>移位的对数转换</strong>表现良好，但不应将其用于每百万计数作为输入，因为它反映了一种不切实际的过度分散。</p>
<p><strong>公共因子</strong>对所有基因进行缩放，人们假设由于细胞大小导致的计数深度差异可以忽略不计，然而对于异质性 scRNA-seq 数据集（<em>数据集由具有非相同细胞特性的不同细胞类型组成</em>），则定义每个基因的统计量可能不准确。</p>
<p><strong>Scran标准化</strong>通过合并具有相似计数深度的细胞，并使用基因的线性回归估计池大小因子，旨在最小化这个问题。</p>
<p><strong>皮尔逊残差</strong>的解析近似，该方法将测序深度作为协变量拟合广义线性模型以获得转换计数矩阵。<mark>归一化方法应根据后续分析任务仔细选择。移位对数被证明在后续降维中更有效地稳定方差，Scran 在批次校正任务中表现良好，解析皮尔逊残差更适合生物变量基因的选择和稀有细胞身份的识别。</mark></p>
</blockquote>
<p>因此我们需要对数据进行标准化操作，使得所有的细胞数据间能够进行比较。</p>
<figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment">#TODO</span></span><br></pre></td></tr></table></figure>
<p>如果数据来源于多个不同的技术重复，那我们应该在分析前（对同一实验处理组内）进行去除批次效应，确保其不会影响我们观察到的生物学现象。</p>
<blockquote>
<p>**主成分分析(PCA)**和 Harmony等线性嵌入模型在具有明显批次结构的简单整合任务中表现出色，适用于批次校正。<strong>scANVI</strong>可以整合细胞类型标签，这有利于保留生物学变异。根据整合任务的复杂性，例如图谱整合，深度学习方法如 <strong>scANVI、scVI</strong>和 <strong>scGen</strong>以及线性嵌入模型如 <strong>Scanorama</strong>表现最佳，而<mark>对于较简单的整合任务，Harmony是首选方法。</mark></p>
</blockquote>
<h4 id="关键"><a class="markdownIt-Anchor" href="#关键"></a> 关键</h4>
<p>先尝试用</p>
<blockquote>
<p>A single-cell rice atlas integrates multi-species data to reveal cis-regulatory evolution</p>
</blockquote>
<p>的数据来对非配对的这个多组学整合进行实验</p>
<p>换几个平台分析从R的<code>seurat</code>换到Python的<code>ScanPy</code></p>
<p>先看几篇文献了解一下</p>
<blockquote>
<ol>
<li>Artificial intelligence for multimodal data integration in oncology</li>
<li>Artificial Intelligence in Molecular Medicine</li>
</ol>
</blockquote>
<p>具体流程以及代码如下：</p>
<h4 id="数据载入"><a class="markdownIt-Anchor" href="#数据载入"></a> 数据载入</h4>
<figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">import</span> </span><br></pre></td></tr></table></figure>
<hr>
<h1 id="reference"><a class="markdownIt-Anchor" href="#reference"></a> Reference</h1>
<p>1.Marand, A. P., Chen, Z., Gallavotti, A. &amp; Schmitz, R. J. A cis-regulatory atlas in maize at single-cell resolution. <em>Cell</em> <strong>184</strong>, 3041-3055.e21 (2021).</p>
<p>2.Marand, A. P. <em>et al.</em> The genetic architecture of cell type–specific cis regulation in maize. <em>Science</em> <a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.ads6601">https://doi.org/10.1126/science.ads6601</a> (2025) doi:<a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.ads6601">10.1126/science.ads6601</a>.</p>
<p>3.Yan, H. <em>et al.</em> A single-cell rice atlas integrates multi-species data to reveal cis-regulatory evolution. <em>Nat. Plants</em> 1–22 (2025) doi:<a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41477-025-02106-6">10.1038/s41477-025-02106-6</a>.</p>
<p>4.Kan, Y. <em>et al.</em> A comparison of integration methods for single-cell RNA sequencing data and ATAC sequencing data. <em>Quantitative Biology</em> <strong>13</strong>, e91 (2025).</p>
<p>5.Lin, Y. <em>et al.</em> scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning. <em>Nat Biotechnol</em> <strong>40</strong>, 703–710 (2022).</p>
<p><a target="_blank" rel="noopener" href="http://6.Ma">6.Ma</a>, S. <em>et al.</em> Chromatin Potential Identified by Shared Single-Cell Profiling of RNA and Chromatin. <em>Cell</em> <strong>183</strong>, 1103-1116.e20 (2020).</p>
<p>7.Zhu, C. <em>et al.</em> An ultra high-throughput method for single-cell joint analysis of open chromatin and transcriptome. <em>Nat Struct Mol Biol</em> <strong>26</strong>, 1063–1070 (2019).</p>
<p>8.Liu, L. <em>et al.</em> Deconvolution of single-cell multi-omics layers reveals regulatory heterogeneity. <em>Nat Commun</em> <strong>10</strong>, 470 (2019).</p>
<p>9.Chen, J., Xiao, W., Zhang, E. &amp; Chen, X. Abstract 4943: Benchmarking unpaired single-cell RNA and single-cell ATAC integration. <em>Cancer Res</em> <strong>84</strong>, 4943 (2024).</p>
<p>10.Fu, S. <em>et al.</em> Benchmarking single-cell multi-modal data integrations. <em>Nat Methods</em> 1–12 (2025) doi:<a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41592-025-02737-9">10.1038/s41592-025-02737-9</a>.</p>
<p>11.Lee, M. Y. Y., Kaestner, K. H. &amp; Li, M. Benchmarking algorithms for joint integration of unpaired and paired single-cell RNA-seq and ATAC-seq data. <em>Genome Biology</em> <strong>24</strong>, 244 (2023).</p>
<p>12.Cao, Z.-J. &amp; Gao, G. Multi-omics single-cell data integration and regulatory inference with graph-linked embedding. <em>Nat Biotechnol</em> <strong>40</strong>, 1458–1466 (2022).</p>
<p>13.Heumos, L. <em>et al.</em> Best practices for single-cell analysis across modalities. <em>Nat Rev Genet</em> <strong>24</strong>, 550–572 (2023).</p>
</div>
        </div>
        
        <footer class="kratos-entry-footer clearfix">
            
            
            
                <div class="post-actions text-center clearfix" id="post-actions">
                
                
                    <a class="share" href="javascript:;" onclick="krOpenShareModal()"><i class="fa fa-share-alt"></i> 分享</a>
                
                </div>
            
            <div class="footer-tag clearfix">
                <div class="pull-left">
                <i class="fa fa-tags"></i>
                    <a class="tag-none-link" href="/myblog.github.io/tags/Multiomics/" rel="tag">Multiomics</a>, <a class="tag-none-link" href="/myblog.github.io/tags/Single-cell/" rel="tag">Single cell</a>
                </div>
                <div class="pull-date">
                    <time datetime="2025-10-22T07:14:55.299Z" itemprop="dateModified">最后编辑：2025-10-22</time>
                </div>
            </div>
        </footer>
    </div>
    
        <nav class="navigation post-navigation clearfix" role="navigation">
            
            <div class="nav-previous clearfix">
                <a title=" 单细胞scRNA-seq分析思路总结" href="/myblog.github.io/posts/单细胞scRNA-seq分析思路总结/"><i class="fa fa-angle-left"></i> 上一篇</a>
            </div>
            
            
            <div class="nav-next clearfix">
                <a title=" Artificial intelligence for multimodal data integration in oncology 文献阅读" href="/myblog.github.io/posts/Artificial-intelligence-for-multimodal-data-integration-in-oncology-文献阅读/">下一篇 <i class="fa fa-angle-right"></i></a>
            </div>
            
        </nav>
    
    
        <div class="comment-post">
            <div id="gitalk-container" class="kr-comments lazy-load" data-path="/myblog.github.io/posts/AI-Multiomics/" data-path-hash="8c028116a8a2ad40cfe6401ea2b6eab6"></div>
        </div>
    
</article>

        

            </section>

            
                

<section id="kratos-widget-area" class="col-md-4 hidden-xs hidden-sm">
    <!-- 文章和页面根据splitter来分割，没有的话就从头开始设置为sticky -->
    
    
                <aside id="krw-about" class="widget widget-kratos-about clearfix">
    <div class="photo-background"></div>
    <div class="photo-wrapper clearfix">
        <div class="photo-wrapper-tip text-center">
            <img class="about-photo" src="/myblog.github.io/images/IMG_5529.GIF" alt="Hammerbrother" loading="lazy" decoding="auto" />
        </div>
    </div>
    <div class="textwidget">
        <p class="text-center">Have a good time!!</p>
    </div>
    <div class="site-meta">
        <a class="meta-item" href="/myblog.github.io/archives/">
            <span class="title">
                文章
            </span>
            <span class="count">
                9
            </span>
        </a>
        <a class="meta-item" href="/myblog.github.io/categories/">
            <span class="title">
                分类
            </span>
            <span class="count">
                4
            </span>
        </a>
        <a class="meta-item" href="/myblog.github.io/tags/">
            <span class="title">
                标签
            </span>
            <span class="count">
                12
            </span>
        </a>
    </div>
</aside>
            
                    <div class="sticky-area">
                
                    <aside id="krw-toc" class="widget widget-kratos-toc clearfix toc-div-class" >
    <div class="photo-background"></div>
    <h4 class="widget-title no-after">
        <i class="fa fa-compass"></i>
        文章目录
        <span class="toc-progress-bar" role="progressbar" aria-label="阅读进度："></span>
    </h4>
    <div class="textwidget">
        <ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#%E5%8D%95%E7%BB%86%E8%83%9E%E5%A4%9A%E7%BB%84%E5%AD%A6%E6%8A%80%E6%9C%AF%E5%AE%9E%E8%B7%B5"><span class="toc-text"> 单细胞多组学技术实践</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#%E7%90%86%E8%AE%BA%E5%88%86%E6%9E%90"><span class="toc-text"> 理论分析</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#%E7%9F%A5%E8%AF%86%E7%82%B9"><span class="toc-text"> 知识点：</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%E9%85%8D%E5%AF%B9%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95"><span class="toc-text"> 配对数据分析方法</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%E9%9D%9E%E9%85%8D%E5%AF%B9%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%E6%96%B9%E6%B3%95%E9%87%8D%E7%82%B9"><span class="toc-text"> 非配对数据分析方法（重点）</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%E8%AF%A5%E9%A2%86%E5%9F%9F%E7%A0%94%E7%A9%B6%E8%BF%9B%E5%B1%95"><span class="toc-text"> 该领域研究进展</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#%E7%9B%AE%E5%89%8D%E6%80%9D%E8%B7%AF"><span class="toc-text"> 目前思路</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#%E6%B5%81%E7%A8%8B"><span class="toc-text"> 流程</span></a><ol class="toc-child"><li class="toc-item toc-level-5"><a class="toc-link" href="#1%EF%B8%8F%E2%83%A3%E6%95%B0%E6%8D%AE%E5%87%86%E5%A4%87"><span class="toc-text"> 1️⃣数据准备</span></a></li></ol></li><li class="toc-item toc-level-4"><a class="toc-link" href="#%E5%85%B3%E9%94%AE"><span class="toc-text"> 关键</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#%E6%95%B0%E6%8D%AE%E8%BD%BD%E5%85%A5"><span class="toc-text"> 数据载入</span></a></li></ol></li></ol></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#reference"><span class="toc-text"> Reference</span></a></li></ol>
    </div>
</aside>
                
                
  <aside id="krw-categories" class="widget widget-kratos-categories clearfix">
    <h4 class="widget-title"><i class="fa fa-folder"></i>分类列表</h4>
      <ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/myblog.github.io/categories/Done/">Done!</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/myblog.github.io/categories/Unfinished/">Unfinished</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/myblog.github.io/categories/new/">new!</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/myblog.github.io/categories/new%EF%BC%81/">new！</a><span class="category-list-count">1</span></li></ul>
  </aside>


            
                
  <aside id="krw-tags" class="widget widget-kratos-tags clearfix">
    <h4 class="widget-title"><i class="fa fa-tags"></i>标签聚合</h4>
      <div class="tag-clouds">
        <a href="/myblog.github.io/tags/AI/" style="font-size: 0.7em;">AI</a> <a href="/myblog.github.io/tags/Multiomics/" style="font-size: 0.7em;">Multiomics</a> <a href="/myblog.github.io/tags/Private/" style="font-size: 0.7em;">Private</a> <a href="/myblog.github.io/tags/Single-cell/" style="font-size: 0.6em;">Single cell</a> <a href="/myblog.github.io/tags/Vim/" style="font-size: 0.6em;">Vim</a> <a href="/myblog.github.io/tags/hexo/" style="font-size: 0.6em;">hexo</a> <a href="/myblog.github.io/tags/kratos/" style="font-size: 0.6em;">kratos</a> <a href="/myblog.github.io/tags/scRNA-seq/" style="font-size: 0.6em;">scRNA-seq</a> <a href="/myblog.github.io/tags/%E4%B8%AA%E4%BA%BA%E7%AC%94%E8%AE%B0/" style="font-size: 0.6em;">个人笔记</a> <a href="/myblog.github.io/tags/%E5%8D%95%E7%BB%86%E8%83%9E/" style="font-size: 0.6em;">单细胞</a> <a href="/myblog.github.io/tags/%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/" style="font-size: 0.6em;">文献阅读</a> <a href="/myblog.github.io/tags/%E8%AE%B0%E5%BD%95/" style="font-size: 0.8em;">记录</a>
      </div>
  </aside>

            
                
  <aside id="krw-posts" class="widget widget-kratos-posts">
  <h4 class="widget-title"><i class="fa fa-file"></i>最新文章</h4>
  <div class="tab-content">
      <ul class="list-group">
        
        
          
          
            <a class="list-group-item" href="/myblog.github.io/posts/Epigenetic-regulation/"><i class="fa fa-book"></i> Epigenetic regulation</a>
            
          
        
          
          
            <a class="list-group-item" href="/myblog.github.io/posts/%E7%A0%94%E7%A9%B6%E6%80%9D%E8%B7%AF/"><i class="fa fa-book"></i> 研究思路</a>
            
          
        
          
          
            <a class="list-group-item" href="/myblog.github.io/posts/Vim%E7%BE%8E%E5%8C%96%E9%85%8D%E7%BD%AE/"><i class="fa fa-book"></i> Vim美化配置</a>
            
          
        
          
          
            <a class="list-group-item" href="/myblog.github.io/posts/Deeplearning-%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0/"><i class="fa fa-book"></i> Artificial intelligence Deeplearning 学习笔记</a>
            
          
        
          
          
            <a class="list-group-item" href="/myblog.github.io/posts/Artificial-intelligence-for-multimodal-data-integration-in-oncology-%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/"><i class="fa fa-book"></i> Artificial intelligence for multimodal data integration in oncology 文献阅读</a>
            
          
        
          
            
          
            
          
            
          
            
      </ul>
  </div>
  </aside>

            
    </div>
</section>
            

        

                </div>
    </div>
</div>

<footer>
    <div id="footer">
        <div class="container">
            <div class="row">
                <div class="col-md-6 col-md-offset-3 footer-list text-center">
                    <ul class="kratos-social-icons">
                        
                            <li>
                                <a href="https://github.com/Hyperion1230" target="_blank" rel="nofollow">
                                    <i class="fa fa-github"></i>
                                </a>
                            </li>
                        
                            <li>
                                <a href="/myblog.github.io/atom.xml" target="_blank" rel="nofollow">
                                    <i class="fa fa-rss"></i>
                                </a>
                            </li>
                        
                            <li>
                                <a href="mailto:1092806479@qq.com" target="_blank" rel="nofollow">
                                    <i class="fa fa-envelope"></i>
                                </a>
                            </li>
                        
                    </ul>
                    <ul class="kratos-copyright">
                        <div>
                            <li>&copy; 2025 Hammerbrother的博客 版权所有.</li>
                            <li>本站已运行<span id="kr-since">Loading...</span></li>
                        </div>
                        <div>
                            <!-- <li>自豪地使用 <a href="https://github.com/Candinya/Kratos-Rebirth" target="_blank">Kratos:Rebirth</a> 主题</li> -->
                            <li>站点由 <a href="https://github.com/Hyperion1230/myblog.github.io/" target="_blank">Hammerbrother</a> 用 <i class="fa fa-heart" style="color:#d43f57"></i> 搭建</li>
                        </div>
                        <!-- 额外的追加注入项 -->
                        
                            <div>
                                
                                    <li>由 <a href="https://hexo.io" target="_blank" rel="nofollow">Hexo</a> 部署</li>
                                
                                    <li><a href="https://github.io" target="_blank">Github Pages</a> 托管</li>
                                
                            </div>
                        
                    </ul>
                </div>
            </div>
        </div>
        <div class="kr-tool text-center">
            <div class="tool">
                
                    <div class="box search-box">
                        <a href="/myblog.github.io/search">
                            <span class="fa fa-search"></span>
                        </a>
                    </div>
                
                <div class="box theme-box" id="theme-toggle">
                    <span class="fa fa-adjust"></span>
                </div>
            </div>
            <div id="gotop-box" class="box gotop-box">
                <span class="fa fa-chevron-up"></span>
            </div>
        </div>
    </div>
</footer>
</div>
</div>


    <div id="kr-donate-modal" class="kr-modal">
    <div class="kr-modal-bg" onclick="krCloseDonateModal()"></div>
    <div class="kr-modal-content">
        <div class="kr-modal-header">
            <div class="kr-modal-icon">
                <i class="fa fa-bitcoin"></i>
            </div>
            <div class="kr-modal-title">赞助喵</div>
            <button class="kr-modal-close" onclick="krCloseDonateModal()">
                <i class="fa fa-times-circle"></i>
            </button>
        </div>
        <div class="kr-modal-body">
            <div class="kr-donate-message">非常感谢您的喜欢！ <i class="fa fa-heart" style="color:#d43f57"></i></div>
            <div class="kr-donate-qr" id="kr-donate-qr">
                
                    <img class="kr-donate-placeholder" src="https://dev.krt.moe/demo-assets/donate.webp" alt="赞助喵" />
                
            </div>
            <div class="kr-donate-platforms">
                
                    
                        <button class="kr-donate-platform-button" style="color: skyblue;" title="模拟支付" onclick="krDonateModalShowPlatformQR('example_payment', 'skyblue')">
                            <i class="fa fa-credit-card"></i>
                        </button>
                    
                
            </div>
        </div>
    </div>
</div>
<script defer src="/myblog.github.io/js/kr-modal/donate.min.js"></script>




    <div id="kr-share-modal" class="kr-modal">
    <div class="kr-modal-bg" onclick="krCloseShareModal()"></div>
    <div class="kr-modal-content">
        <div class="kr-modal-header">
            <div class="kr-modal-icon">
                <i class="fa fa-share-alt"></i>
            </div>
            <div class="kr-modal-title">Share this moment！</div>
            <button class="kr-modal-close" onclick="krCloseShareModal()">
                <i class="fa fa-times-circle"></i>
            </button>
        </div>
        <div class="kr-modal-body">
            <div class="kr-share-message">好东西要和大家分享！</div>
            <div class="kr-share-qr" id="kr-share-qr"></div>
            <div class="kr-share-platforms">
                
                    <button class="kr-share-platform-button" style="color: #62b6e7;" title="喵窝" onclick="krShareModalOpenPlatform('https://nya.one/share?text=%3E%20$TITLE%0A%3E%0A%3E%20$SUMMARY%0A----%20$SITE%0A%0A$URL')">
                        <svg fill="currentColor" width="18" height="18" viewBox="0 0 329 388"><path d="M0,36.226c0,-19.993 16.233,-36.226 36.226,-36.226c13.788,0 25.787,7.719 31.908,19.068l188.366,229.845l0,-111.259c-6.356,5.508 -14.645,8.843 -23.707,8.843c-19.994,0 -36.226,-16.232 -36.226,-36.226c-0,-8.551 2.968,-16.413 7.93,-22.613l-0.015,-0.013l56.614,-69.08c6.2,-11.073 18.047,-18.565 31.63,-18.565c19.994,0 36.227,16.233 36.227,36.226l-0,314.644c-0,19.994 -16.233,36.226 -36.227,36.226c-13.166,0 -24.701,-7.039 -31.044,-17.555l-189.229,-230.899l-0,212.228c-0,19.994 -16.233,36.226 -36.227,36.226c-19.993,0 -36.226,-16.232 -36.226,-36.226l0,-314.644Z" /></svg>
                    </button>
                
                    <button class="kr-share-platform-button" style="color: #12B7F5;" title="QQ" onclick="krShareModalOpenPlatform('https://connect.qq.com/widget/shareqq/index.html?url=$URL&title=$TITLE&desc=&summary=$SUMMARY&site=$SITE')">
                        <i class="fa fa-qq"></i>
                    </button>
                
                    <button class="kr-share-platform-button" style="color: #07c160;" title="微信" onclick="krShareModalOpenPlatform('https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=$URL')">
                        <i class="fa fa-weixin"></i>
                    </button>
                
            </div>
        </div>
    </div>
</div>
<script defer src="/myblog.github.io/js/kr-modal/share.min.js"></script>



<!-- 额外的追加注入项 -->

    <script>
(function() {
  var hour = new Date().getHours();
      var isDark = hour >= 18 || hour < 6;
        if (isDark) {
          document.documentElement.setAttribute('data-theme', 'dark');
      } else {
          document.documentElement.setAttribute('data-theme', 'light');
                    }
                    })();
                    </script>
<meting-js server="netease" type="playlist" id="12511958644" fixed="true" order="random"></meting-js>
                        



        <script>
    if (!window.kr) {
        window.kr = {};
    }
    window.kr.notMobile = (!(navigator.userAgent.match(/(phone|pad|pod|iPhone|iPod|ios|iPad|Android|Mobile|BlackBerry|IEMobile|MQQBrowser|JUC|Fennec|wOSBrowser|BrowserNG|WebOS|Symbian|Windows Phone)/i)));
    window.kr.siteRoot = "/myblog.github.io/";
</script>


  <script defer src="/myblog.github.io/vendors/nprogress@0.2.0/nprogress.js"></script>



  <script defer src="/myblog.github.io/vendors/viewerjs@1.11.6/dist/viewer.min.js"></script>


<script defer src="/myblog.github.io/js/kr-core.min.js"></script>


  <script defer src="/myblog.github.io/js/kr-pjax.min.js"></script>


<!-- 额外的追加注入项 -->

  <script src="https://unpkg.com/gitalk/dist/gitalk.min.js"></script>
<script src="/myblog.github.io/comments/gitalk.js"></script>
<script src="/myblog.github.io/music/APlayer.min.js"></script>
<script src="/myblog.github.io/music/Meting.min.js"></script>



    </body>
</html>